packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -160,14 +161,15 @@ export class BlockSwitcher extends Component {
 						}
 
 						{ ( hoveredClassName !== null ) &&
-							<BlockPreview
-								blocks={
-									cloneBlock( blocks[ 0 ], {
-										className: hoveredClassName,
-									} )
-								}
-
-							/>
+							<BlockPreviewDropdownWrapper>
```

Comment:
Using a `<BlockPreviewDropdownWrapper /> `and `< BlockPreview />` looks slightly complicated unnecessarily. The @youknowriad suggestion of adding a generic `PopoverAside` component could justify this wrapper but as I mentioned above it could require more time and also is kind of out of the scope imo.

What do you think to handle the anchoring inside of the <PreviewBlock /> through of a new property?

We can iterate later if we want to implement the aside mode component.

Reply:
I  don't think BlockPreview component should concern  itself about anchoring. It should basically be just a simple component that shows a preview of blocks. Any UI outside that (wrapper, popover anchor...) is IMO a separate concern. I don't mind it to be duplicated in the caller if necessary.

Reply:
> It should basically be just a simple component that shows a preview of blocks.

Fair enough.